nitro-node/.gitignore

```diff
@@ -0,0 +1,23 @@
+.vscode
+.env
+
+# Jan inference
+error.log
+.yarn
+node_modules
+*.tgz
+yarn.lock
+dist
+build
+.DS_Store
+package-lock.json
+
+*.log
+
+# Nitro binary files
+bin/*/nitro
+bin/*/*.metal
+bin/*/*.exe
+bin/*/*.dll
+bin/*/*.exp
+bin/*/*.lib
```

Comment:
What files should we keep?

Reply:
We don't need them in sourcecode, as there is already a `postinstall` hook in `package.json` that will download all the binaries into the correct place if `@janhq/nitro-node` is installed as a dependency. I tested with both npm and yarn@1 and that works.

Reply:
So pattern ignore should work? e.g. bin or bin*/*.*

Reply:
I think just need to ignore the whole `bin` directory. Let me update the ignore file.